sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -593,8 +593,9 @@ object GpuOverrides extends Logging {
   }
 
   def isSupportedStringReplacePattern(strLit: String): Boolean = {
-    // check for regex special characters, except for \u0000 which we can support
-    !regexList.filterNot(_ == "\u0000").exists(pattern => strLit.contains(pattern))
+    // check for regex special characters, except for \u0000, \n, \r, and \t which we can support
```

Comment:
nit:

https://github.com/NVIDIA/spark-rapids/blob/b48335ab78ae2b3056d7063aeae7916c6dbd53e5/sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala#L609

Reply:
Deleted this comment as it is duplicated and intended to describe the internal logic of this function.